lib/core/types.hpp

```diff
@@ -43,7 +44,7 @@ using EncoderData          = std::array<std::uint32_t, kNumEncoders>;
 using KeyenceData          = std::array<std::uint32_t, kNumKeyence>;
 
 // data produced by the accelerometer sensor
-// !! the values are in mg !!
+// values are in mg
```

Comment:
kinda picky but i would prefer to distinguish between milli g and milligram

Reply:
i mean, the accelerometer surely wont be giving grams as units

Reply:
it caught me out for a few seconds but maybe im just dumb

Reply:
To be fair, mg is the SI notation for milligrams so you might as well use the full words for clarity (in case someone reding the code isn't familiar with accelerometers)?

Reply:
I highly doubt that but will change